include/glowl/GLSLProgram.hpp

```diff
@@ -197,38 +206,23 @@ namespace glowl
         glCompileShader(shader);
 
         /* Check for errors */
-        GLint compile_ok = GL_FALSE;
-        glGetShaderiv(shader, GL_COMPILE_STATUS, &compile_ok);
-        if (compile_ok == GL_FALSE)
-        {
-            GLint logLen = 0;
-            m_shaderlog = "";
-            glGetShaderiv(shader, GL_INFO_LOG_LENGTH, &logLen);
-            if (logLen > 0)
-            {
-                char* log = new char[logLen];
-                GLsizei written;
-                glGetShaderInfoLog(shader, logLen, &written, log);
-                m_shaderlog = log;
-                delete[] log;
+        GLint compileStatus = GL_FALSE;
```

Comment:
Nitpicking incoming: Can we change the variable naming style to snake case (*_*) for consistency we the rest of the code?